README.md

```diff
@@ -175,12 +175,6 @@ steps:
 
 > Note: The `id` defined in `actions/cache` must match the `id` in the `if` statement (i.e. `steps.[ID].outputs.cache-hit`)
 
-## Known limitation
-
-- `action/cache` is currently not supported on GitHub Enterprise Server. <https://github.com/github/roadmap/issues/273> is tracking this.
-
-Since GitHub Enterprise Server uses self-hosted runners, dependencies are typically cached on the runner by whatever dependency management tool is being used (npm, maven, etc.).  This eliminates the need for explicit caching in some scenarios.
-
 ## Changelog schedule and history 
 
 | Status  | Version  | Date  | Highlights  |
```

Comment:
Add to this changelog with <todo> version number which can be updated when publishing.
Also move this changelog to a separate file named `CHANGELOG`

Reply:
Created a file with the name RELEASE.md because found this nomenclature is used toolkit as well.